File: 1534-count-good-triplets/1534-count-good-triplets.py
class Solution:
    def countGoodTriplets(self, arr: List[int], a: int, b: int, c: int) -> int:
        count = 0
        n = len(arr)
        for i in range(0,n-2):
            for j in range(i+1,n-1):
                for k in range(j+1,n):
                    x = abs(arr[i] - arr[j])
                    y = abs(arr[j] - arr[k]) 
                    z = abs(arr[i] - arr[k])       
                    if x <= a and y <= b and z <= c:
                        count += 1
        return count

File: 1534-count-good-triplets/1534-count-good-triplets.java
class Solution {
    public int countGoodTriplets(int[] arr, int a, int b, int c) {
        int total = 0;
        for (int i = 0; i < arr.length-2;i++){
            for (int j = i+1; j < arr.length-1;j++){
                for (int k = j+1; k < arr.length;k++){
                    boolean exp1 = Math.abs(arr[i] - arr[j]) <= a;
                    boolean exp2 = Math.abs(arr[j] - arr[k]) <= b;
                    boolean exp3 = Math.abs(arr[i] - arr[k]) <= c;
                    total += exp1 && exp2 && exp3 ? 1 : 0;
                }
            }
        }
        return total; 
    }
}